src/main/java/org/eclipse/tractusx/puris/backend/common/api/domain/repository/ResponseRepository.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 Volkswagen AG
+ * Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.eclipse.tractusx.puris.backend.common.api.domain.repository;
+
+import org.eclipse.tractusx.puris.backend.common.api.domain.model.Request;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.UUID;
+
+/**
+ * Repository to access Responses
+ */
+public interface ResponseRepository extends JpaRepository<Request, UUID> {
+
+    /**
+     * find the request by the requestUuuid from the message's header
```

Comment:
JavaDoc comments should start with a capital letter. But this can be fixed in #31.